### DISCUSSION TITLE
Is there a way to snap objects to a grid like Gimp?

### DISCUSSION THREAD

**Discussion by x0rb** [excalidraw/excalidraw#7862] [General]
Is there a way:

- display a grid
- snap object to the grid

Goal is to evenly distribute objects.

eg., https://docs.gimp.org/2.10/en/gimp-view-snap-to-grid.html

See also: https://github.com/excalidraw/excalidraw/discussions/5345#discussion-4164359

**Comment by Lebohang1821** [comment]
Go to View > Show Grid.

   To snap objects to the grid: After enabling the grid, when you move objects, they will automatically snap to the grid lines if the "Snap to Grid" option is enabled. You can toggle this option under View > Snap to Grid.
